climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1204 @@
+            # within each country and combined at the end.
+            litpop_list = \
+                [self._set_one_country(country,
+                                       res_arcsec=res_arcsec,
+                                       exponents=exponents,
+                                       fin_mode=fin_mode,
+                                       total_value=total_values[idc],
+                                       reference_year=reference_year,
+                                       gpw_version=gpw_version,
+                                       data_dir=data_dir,
+                                       reproject_first=reproject_first)
+                 for idc, country in enumerate(countries)]
+        # make lists of countries with Exposure initaited and those ignored:
+        countries_in = \
+            [country for i, country in enumerate(countries) if litpop_list[i] is not None]
+        countries_out = \
+            [country for i, country in enumerate(countries) if litpop_list[i] is None]
+        if not countries_in:
+            raise ValueError('No valid country identified in %s, aborting.' % countries)
+        litpop_list = [exp for exp in litpop_list if exp is not None]
+        if countries_out:
+            LOGGER.warning('Some countries could not be identified and are ignored: ' +
+                           '%s. Litpop only initiated for: %s', countries_out, countries_in)
+
+        tag.description = ('LitPop Exposure for %s at %i as, year: %i, financial mode: %s, '
+                           'exp: [%i, %i], admin1_calc: %s'
+                           % (countries_in, res_arcsec, reference_year, fin_mode,
+                              exponents[0], exponents[1], str(admin1_calc)))
+
+        Exposures.__init__(
+            self,
+            data=Exposures.concat(litpop_list).gdf,
+            crs=litpop_list[0].crs,
+            ref_year=reference_year,
+            tag=tag,
+            value_unit=get_value_unit(fin_mode),
+            exponents = exponents,
+            gpw_version = gpw_version,
+            fin_mode = fin_mode,
+        )
+
+        try:
+            rows, cols, ras_trans = u_coord.pts_to_raster_meta(
+                (self.gdf.longitude.min(), self.gdf.latitude.min(),
+                 self.gdf.longitude.max(), self.gdf.latitude.max()),
+                u_coord.get_resolution(self.gdf.longitude, self.gdf.latitude))
+            self.meta = {
+                'width': cols,
+                'height': rows,
+                'crs': self.crs,
+                'transform': ras_trans,
+            }
+        except ValueError:
+            LOGGER.warning('Could not write attribute meta, because exposure'
+                           ' has only 1 data point')
+            self.meta = {}
+        self.check()
+
+    def set_nightlights(self, countries=None, shape=None, res_arcsec=15,
+                        reference_year=2016, data_dir=SYSTEM_DIR):
```

Comment:
reference_year in this function defaults to 2016, whereas in all other functions it defaults to 2020.
Maybe import `DEF_REF_YEAR` from `climada.entity.exposures.base` and use it as default in all litpop.set_* functions. Or even better create a def_ref_year in the config file for all exposures and set it to 2020

Reply:
i have put it to config: https://github.com/CLIMADA-project/climada_python/pull/239/commits/7c203ef850141e69b02bc3a4eac5247474d89cd2, https://github.com/CLIMADA-project/climada_python/pull/239/commits/976adeb8fc0abe853d4a544d90d5309e6d1cb963